solutions/instances/variables.tf

```diff
@@ -123,6 +123,12 @@ variable "cloud_monitoring_provision" {
   default     = true
 }
 
+variable "existing_cloud_monitoring_crn" {
+  description = "The CRN of existing IBM Cloud Monitoring instance"
+  type        = string
+  default     = null
+}
```

Comment:
I will suggest to add in description like if it is set or provided a value, then var cloud_monitoring_provision should be false. Because by default we set that to `true`.

Also, good to validate the same in locals.

Reply:
addressed